Xamarin.Forms.Platform.MacOS/Renderers/SliderRenderer.cs

```diff
@@ -1,10 +1,70 @@
 ﻿using System;
 using AppKit;
 using System.ComponentModel;
-using Xamarin.Forms.Platform.macOS.Controls;
+using CoreGraphics;
 
 namespace Xamarin.Forms.Platform.MacOS
 {
+	public class XFSliderCell : NSSliderCell
```

Comment:
I think wee don't need this to be public.

Reply:
done